src/modules/student/dto/create-student.dto.ts

```diff
@@ -0,0 +1,30 @@
+import { ContractType, NewStudentEntity, TypeWork } from 'src/types';
+
+export class CreateStudentDto implements NewStudentEntity {
+  id: string;
+  avatar: string | null;
+  bio: string;
+  bonusProjectUrls: string[];
+  canTakeApprenticeship: boolean;
+  courseCompletion: number;
+  courseEngagement: number;
+  courses: string | null;
+  education: string | null;
+  email: string;
+  expectedContractType: ContractType;
+  expectedSalary: number | null;
+  expectedTypeWork: TypeWork;
+  firstName: string;
+  githubUsername: string;
+  lastName: string;
+  monthsOfCommercialExp: number;
+  portfolioUrls: string[] | null;
+  projectDegree: number;
+  projectUrls: string[];
+  targetWorkCity: string;
+  teamProjectDegree: number;
+  tel: string | null;
+  workExperience: string | null;
+  status: number;
+  [key: string]: any;
```

Comment:
żeby nie było any to trzeba zrobić [key: string]: i tu wypisać wszystkie typy jakie występują w encji

Reply:
ok, tak zrobię ;)